source/unified-test-format/tests/Makefile

```diff
@@ -1,8 +1,8 @@
-SCHEMA=../schema-1.4.json
+SCHEMA=../schema-1.5.json
 
-.PHONY: all invalid valid-fail valid-pass versioned-api load-balancers gridfs transactions crud collection-management HAS_AJV
+.PHONY: all invalid valid-fail valid-pass versioned-api load-balancers gridfs transactions crud collection-management command-monitoring HAS_AJV
```

Comment:
I've love to find a different option for this at some point in time. Ideally, we'd go through all directories, grab every `.yml` file, attempt to read a schema version for it (skipping the file if we don't find one) and validating the file against the explicit schema version indicated in the file. @jmikola do you think this is feasible? If so I'd file a ticket so we can eventually return to this.

This solution would fix two issues:
1. it would no longer require us to manually add new folders with unified test files here, and
2. it validates files against the schema version they claim to be, which can catch errors when somebody adds functionality from newer schema versions without bumping the `schemaVersion` in the file.

Reply:
[DRIVERS-1767](https://jira.mongodb.org/browse/DRIVERS-1767) already exists to track this. If there's some variation to the behavior (e.g. skipping files that don't have a `schemaVersion` field) feel free to note that as a suggested approach in the ticket.

Reply:
I'll check it out, thanks for pointing me in the right direction